### ISSUE TITLE
Специальный режим поиска по стенограммам В–О

### ISSUE THREAD

**Issue by iprst** [terratensor/svodd-server#11]
Планируется ли реализовать? Дабы клиент, желающий искать только в тексте стенограмм, мог выбрать соответствующую кнопку или пункт меню.

**Comment by audetv** [comment]
В процессе реализации парсера эта мысль возникала, но отбрасывал так сказать, до времен, держал в фоне. Технически для этого нет препятствий и фундамент заложен. Можно реализовать. Надо подумать как это сделать, точнее как оформить для клиента. Можно сделать галочку в настройках поиска (это можно сделать независимо от дополнительных вариантов), но есть предположение, что этого будет не достаточно, возможно, надо сделать что-то подобное, как на сайте поиск по kremlin, mil, mid . сделать метки цветные?

**Comment by audetv** [comment]
> То есть речь о варианте режима «поиска по ФКТ», где «стенограммы» и «комментарии» разделены цветовым маркером? Интересная идея, кажется так и следует думать далее.

Да, на feed выглядит приятно и вроде не мешает просмотру.  Привычка конечно, здесь больше влияет, но ощущение, что метод рабочий. Цвет дает некоторую изюминку. Красный СВОДД будет концептуально выглядеть отлично.

> А комментарии СВОДД и остального сайта чем-то отделены, может эту классификацию тоже разместить в светофор? Было бы «вопрос-ответ», «фкт» и «сводд», или это слишком?

СВОДД отделен от сайта, номерами вопросов, т.е. да, можно отфильтровать только по вопросам и комментариям в СВОДД. Это реально. И номера вопросов пополняются с каждой новой темой, когда задаем следующую тему, каждый комментарий связан со своим вопросом. по этому связующему номеру можно сделать фильтр.

Все, что не попало в СВОДД, будет в ФКТ. Вопрос-ответ имеет свой специальный тип в БД, по этому типу легко отфильтровать. 

Так что да можно сделать светофор, будет свежее и поновее, но самое главное, будет функционал. Я думаю это будет пользоваться спросом. 
Выбираем цвета для маркеров.

**Comment by audetv** [comment]
Еще есть комментарии к вопрос ответ, к стенограмме, думаю , что такие комментарии, могут быть комментариями фкт. Кажется, что нет смысла их выделять как то отдельно. Нужно ли фильтровать и выдавать их в поисковой выдаче вместе с чанками вопрос ответ? Думаю, что нет. Но технически эти комментарии тоже можно различить, но кажется, что не нужно.

**Comment by iprst** [comment]
> Выбираем цвета для маркеров.

Так? В наших уже используемых цветах.

![image](https://github.com/user-attachments/assets/a380aa5d-ee63-4b55-a34f-650e71464274)

**Comment by iprst** [comment]
> Еще есть комментарии к вопрос ответ, к стенограмме, думаю , что такие комментарии, могут быть комментариями фкт.

Да, тоже думаю это должны быть комментарии фкт, и что отдельно их выделять из общей ленты комментариев не следует.
